modules/API/API.js

```diff
@@ -257,6 +257,14 @@ export default {
     },
 
     /**
+     * Sends remote control event.
```

Comment:
Can you provide more details about this method and it's arguments ? What structure does the event have or where does it's value come from ?